src/app/api/tasks/tasks.service.ts

```diff
@@ -2,23 +2,17 @@ import { CreateTaskRequest, UpdateTaskRequest } from '@/types/dto/tasks.dto'
 import { BaseService } from '@api/core/services/base.service'
 import { Resource } from '@api/core/types/api'
 import { PoliciesService } from '@api/core/services/policies.service'
-import { ActivityType, StateType, AssigneeType, Task } from '@prisma/client'
+import { ActivityType, StateType, AssigneeType } from '@prisma/client'
 import { UserAction, UserRole } from '@api/core/types/user'
-import { CopilotAPI } from '@/utils/CopilotAPI'
 import { NotificationTaskActions } from '@api/core/types/tasks'
-import {
-  getEmailDetails,
-  getInProductNotificationDetails,
-  getNotificationParties,
-  getTaskTimestamps,
-} from '@api/tasks/tasks.helpers'
+import { getTaskTimestamps } from '@api/tasks/tasks.helpers'
 import APIError from '@api/core/exceptions/api'
 import httpStatus from 'http-status'
 import { ActivityLogger } from '@api/activity-logs/services/activity-logger.service'
 import { TaskCreatedSchema } from '@api/activity-logs/schemas/TaskCreatedSchema'
 import { TaskAssignedSchema } from '@api/activity-logs/schemas/TaskAssignedSchema'
 import { WorkflowStateUpdatedSchema } from '@api/activity-logs/schemas/WorkflowStateUpdatedSchema'
-import { z } from 'zod'
+import { NotificationService } from '../notification/notification.service'
```

Comment:
make alias imports to absolute path

Reply:
Fixed in 3a9dcb86db254fe7b133359ecb68d8b0b1d765be commit